android/src/main/kotlin/im/nfc/flutter_nfc_kit/FlutterNfcKitPlugin.kt

```diff
@@ -360,10 +643,18 @@ class FlutterNfcKitPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
                     tag.techList.contains(MifareClassic::class.java.name) -> {
                         standard = "ISO 14443-3 (Type A)"
                         type = "mifare_classic"
+                        isMifareClassic = true
+                        mifareClassType = mifareClassicGetType()
```

Comment:
I suggest adding another level to reduce the number of possible top-level attributes, like

```
mifareInfo: {
    size: xx,
    sectorCount: xxx
}
```

What do you think @dangfan ?